### PR TITLE
Get rid of another Type.GetTypeCode usage

### PR DESCRIPTION
**Pull request by moodmosaic**
Similar to (merged) Pull Request #518. See also the discussion at #513.

### CODE REVIEW

File: Src/AutoFixture/RandomRangedNumberGenerator.cs [hunk 1]
```diff
@@ -135,13 +133,18 @@ private static long ConvertLimit(object limit)
...
-
-               
             }
 
             throw new ArgumentException("Limit parameter is non-numeric ", "limit");         
-        }      
-       
+        }
+
+        private static TypeCode GetTypeCode(object request)
```
Comment (moodmosaic) [inline Src/AutoFixture/RandomRangedNumberGenerator.cs]: We can move this to a separate file but, according to the [rule of three](http://c2.com/cgi/wiki?RuleOfThree), we can wait.

File: Src/AutoFixture/RandomRangedNumberGenerator.cs [hunk 2]
```diff
@@ -135,13 +133,18 @@ private static long ConvertLimit(object limit)
 
                 case TypeCode.UInt64:
                         return (long)(ulong)limit;
-
-               
             }
 
             throw new ArgumentException("Limit parameter is non-numeric ", "limit");         
-        }      
-
```
Comment (moodmosaic) [inline Src/AutoFixture/RandomRangedNumberGenerator.cs]: I removed a few unnecessary blank lines here and there, by following [the boy scout rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule).

**Comment by ploeh** [comment]
Thank you for your contribution! Since there's no changed behaviour, I haven't released a new version of AutoFixture.

**Comment by moodmosaic** [comment]
Sure, that makes sense. :+1:

**Comment by lbargaoanu** [comment]
A little late, but there is actually Convert.GetTypeCode(object) for both frameworks.

**Comment by ploeh** [comment]
I wouldn't mind a pull request with a further refactoring towards `Convert.GetTypeCode(object)`, if anyone feels like it :smile:

**Pull request merged**